src/org/zaproxy/zap/extension/alert/ExtensionAlert.java

```diff
@@ -376,6 +380,23 @@ private void writeAlertToDB(Alert alert, HistoryReference ref) throws HttpMalfor
 
     }
 
+    private TreePath findAlertInTree(Alert alert) {

```

Comment:
Oh I thought you meant there was some simple call already there that I was missing, I didn't expect you to write code for me....thanks!